.github/workflows/build-android.yml

```diff
@@ -0,0 +1,90 @@
+name: 🤖 Android APK Build
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [develop]
+  push:
+    branches: [develop]
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    name: 🏗️ Build Android APK
+    permissions:
+      contents: read
+      actions: write
+      issues: write
+
+    steps:
+      - name: 📥 Checkout code
+        uses: actions/checkout@v4.2.2
+        with:
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: 🍞 Setup Bun
+        uses: oven-sh/setup-bun@v2.0.2
+        with:
+          # @todo: update to 1.x once this is fixed: https://github.com/streamyfin/streamyfin/pull/690#discussion_r2089749689  
+          bun-version: '1.2.13'
+
+      - name: ☕ Setup JDK
+        uses: actions/setup-java@v4.7.1
+        with:
+          distribution: 'zulu'
+          java-version: '17'
+
+      - name: 📦 Install dependencies
+        run: |
+          bun install --frozen-lockfile
+          bun run submodule-reload
+      
+      - name: 💾 Cache Android prebuild
+        uses: actions/cache@v4
+        with:
+          key: ${{ runner.os }}-bun-${{ hashFiles('bun.lock') }}
+          path: |
+            ./android/
+
+      - name: 🛠️ Generate project files
+        run: bun run prebuild
+
+      - name: 🚀 Build APK via Bun
+        run: bun run build:android:local
+
+      - name: 📤 Upload APK artifact
+        uses: actions/upload-artifact@v4.6.2
+        with:
+          name: streamyfin-apk-${{ github.sha }}
+          path: |
+            android/app/build/outputs/apk/release/*.apk
+            android/app/build/outputs/bundle/release/*.aab
+          retention-days: 7
+
+      - name: 💬 Notify PR
+        uses: actions/github-script@v7.0.1
+        with:
```

Comment:
```suggestion
      - name: 💬 Notify PR
        uses: actions/github-script@v7.0.1
        continue-on-error: true
        with:
```

Reply:
to avoid red mark due to fork PR, we will handle this later, I have an idea to test based on sticky-comment we use on pr-title